website/docs/getting_started/getting_started.md

```diff
@@ -0,0 +1,159 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+slug: /getting_started
+---
+
+## Purchase a Radio
+
+The easiest way is to [buy a device with the software already installed](https://www.aliexpress.com/item/4001178678568.html) In the Americas get the 915mhz version, In Europe the 868Mhz or Asia 923Mhz see [here](https://www.thethingsnetwork.org/docs/lorawan/frequencies-by-country.html).
+
+## Setup the Radio
+
+When it arrives, install your antenna and make sure you install the battery correctly. Reversing the battery can damage your device. Make sure the antenna is on when you power up the board.
+
+:::caution
+Make sure not to power the radio on without first attaching the antenna! You could damage the radio chip!
+:::
+
+## Download Firmware
+
+Prebuilt binaries for the supported radios are available in our releases. Your initial installation has to happen over USB from your Mac, Windows or Linux PC. Once our software is installed, all future software updates happen over bluetooth from your phone.
+
+<!--- TODO I'd like to create prettier buttons for this than just a table --->
+| [Current Firmware](https://github.com/meshtastic/meshtastic-device/releases/latest) | [List of Firmware Versions](https://github.com/meshtastic/meshtastic-device/releases/) |
+| :--------------: | :-----------------------: |
+
+## Flashing Firmware
+
+If your device already has Meshtastic flashed to it, You can update over the air (OTA). Otherwise you'll need a computer and a **data** USB cable.
+
+- [Install using the command line interface (CLI)](getting_started/flashing_firmware/cli)
+- [Install using a graphical user interface (GUI)](getting_started/flashing_firmware/gui)
+- [Install using over-the-air (OTA)](getting_started/flashing_firmware/ota)
+
+## Connect to your Device
+
+There are many ways to connect to your new radio! Using the command line interface
+
+- Command line interface (CLI)
+- Graphic user interface (GUI)
+- Serial connection
```

Comment:
Add android app?

Reply:
Oh of course! I kinda included it in bluetooth. I have another pending PR that will fix this section up better. Maybe I'll add mobile, this way when the iOS app is released we can have the mobile page direct to each App Store.